src/api/app/views/webui/user/_basic_info.html.haml

```diff
@@ -48,18 +48,21 @@
   %p= render_as_markdown(user.biography)
 
   .mt-4
-    - if User.session
+    - if User.session && user.email.present?
       = mail_to(user.email, title: "Send mail to #{user.name}", class: 'd-block') do
         %i.fas.fa-envelope.me-1
         = user.email
 
-  - if Flipper.enabled?(:content_moderation, User.possibly_nobody)
+  - if user != User.possibly_nobody && Flipper.enabled?(:content_moderation, User.possibly_nobody)
     = render partial: 'webui/shared/report_modal'
     - if User.possibly_nobody.is_admin? || User.possibly_nobody.is_moderator?
-      .mt-2
-        This user is currently #{user.blocked_from_commenting ? 'blocked' : 'unblocked'} from commenting.
+      .d-flex.justify-content-between.mt-2.border-top.pt-3
+        .mx-2
+          This user is currently
+          %b= user.blocked_from_commenting ? 'blocked' : 'unblocked'
+          from commenting.
```

Comment:
Sounds like a [bootstrap switch](https://getbootstrap.com/docs/5.3/forms/checks-radios/#switches) to me...

Reply:
To me too: https://github.com/openSUSE/open-build-service/pull/15586#discussion_r1477901120

Reply:
https://github.com/openSUSE/open-build-service/pull/15684/commits/836af640ce07532e921b1dffb9fa057ffa90ea6e

![image](https://github.com/openSUSE/open-build-service/assets/7080830/2a0aa89b-e9b1-4f93-a33a-1d67f3e21f63)